client/gutenberg/extensions/related-posts/style.scss

```diff
@@ -1,3 +1,7 @@
+// @TODO: Replace with Gutenberg variables
+$dark-gray-300: #6c7781;
```

Comment:
Could we use just a tad bit darker gray here to pass [WCAG AAA accessibility test](https://webaim.org/resources/contrastchecker/?fcolor=6C7781&bcolor=FFFFFF)?

Reply:
I'm not entirely sure that's necessary. I think the minimum for a11y on WordPress is a 4.5 contrast ratio meeting the WCAG AA standard. It currently does with this grey.